pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -5,33 +5,36 @@
 The grids are available in various resolutions.
 """
 from pygmt.datasets.load_remote_dataset import _load_remote_dataset
+from pygmt.exceptions import GMTInvalidInput
 from pygmt.helpers import kwargs_to_strings
 
 __doctest_skip__ = ["load_earth_magnetic_anomaly"]
 
 
 @kwargs_to_strings(region="sequence")
 def load_earth_magnetic_anomaly(
-    resolution="01d", region=None, registration=None, mag4km=False
+    resolution="01d", region=None, registration=None, mag4km=False, wdmam=False
```

Comment:
For consistency with `load_earth_relief`, would it be better to use a single `data_source` parameter with 3 options (`emag`, `emag4km`, `wdmam`)? Names to be decided of course.